packages/dx-react-grid/demos/featured/controlled-mode.md

```diff
@@ -1,6 +1,5 @@
 # React Grid Controlled State Mode
 
-This demo shows the Grid capability to delegate its UI state management to the parent component. You can contol either the whole Grid state or just a part of it. In the
-first case, Grid becomes a stateless component. This mode is usefull if you need to use the Grid state in other parts of your application. Using this mode, you can also persist the component state and restore it when required.
+This demo demonstrates the Grid's functionality in the [Controlled State mode](../../docs/guides/controlled-and-uncontrolled-modes.md). That is the parent component manages the whole Grid's state or just its part. In the first case, the Grid works as a stateless component. Use the Controlled State mode when you need to share the Grid's state between other application parts. It also allows you to persist the Grid's state and restore it when required.
```

Comment:
"That is the parent component manages the whole..." This part doesn't make sense. Can you explain it, please?